src/main/java/org/opentripplanner/api/resource/GraphMetadata.java

```diff
@@ -13,22 +13,30 @@
 
 package org.opentripplanner.api.resource;
 
+import java.io.Serializable;
 import java.util.HashSet;
 
 import javax.xml.bind.annotation.XmlElement;
 import javax.xml.bind.annotation.XmlRootElement;
 
 import org.opentripplanner.routing.core.TraverseMode;
-import org.opentripplanner.routing.edgetype.PatternHop;
-import org.opentripplanner.routing.graph.Edge;
 import org.opentripplanner.routing.graph.Graph;
 import org.opentripplanner.routing.graph.Vertex;
 
 import com.vividsolutions.jts.geom.Coordinate;
 import com.vividsolutions.jts.geom.Envelope;
 
+/**
+ * GraphMetada is first created after OSM is build.
+ *
+ * It has two envelopes. One with all of OSM vertices. This is used in {@link org.opentripplanner.graph_builder.module.StreetLinkerModule} and
+ * {@link org.opentripplanner.graph_builder.module.TransitToTaggedStopsModule} to skip stops which are outside OSM data area.
```

Comment:
On second thought, it could be useful to have an OSM-only envelope. But we should probably remove the references in the docs and code to this envelope being used in the stop linker modules.